src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write code here
+// table dom element
+const tableElement = document.querySelector('.field');
+// buttons
+const addRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const addColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+
+const table = {
+  width: 4,
+  height: 4,
+  minSize: 2,
+  maxSize: 10,
+};
+
+addRowBtn.addEventListener('click', (e) => {
+  tableElement.append(tableElement.rows[tableElement.rows.length - 1].cloneNode(true));
```

Comment:
This method will copy also the content of each cell, can you try reseting the inner contend of each cell?